src/AppBundle/Entity/LegislativeCandidate.php

```diff
@@ -241,4 +251,14 @@ public function setEmailAddress(?string $emailAddress): void
     {
         $this->emailAddress = $emailAddress;
     }
+
+    public function getCareer(): ?string
+    {
+        return $this->career;
+    }
+
+    public function setCareer(?string $career)
```

Comment:
I dont get it.
If you want if nullable: you have to set `@ORM\Column(nullable=true)`
If not, you should remove the `?`